includes/misc-functions.php

```diff
@@ -2208,58 +2208,6 @@ function give_get_formatted_address( $address = array() ) {
 	return $formatted_address;
 }
 
-/**
- * Converts a PHP date format for use in JavaScript.
- *
- * @since 2.2.0
- *
- * @param string $php_format The PHP date format.
- *
- * @return string The JS date format.
- */
-function give_convert_php_date_format_to_js( $php_format ) {
```

Comment:
@jaydeeprami we cannot remove function like this add them to deprecate functions file if there is no use of them.